t/benchmark/multi_consumer_work.lua

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env tarantool
+
+local clock = require('clock')
+local os = require('os')
+local fiber = require('fiber')
+local queue = require('queue')
+
+-- Set the number of consumers.
+local consumers_count = 100
+-- Set the number of tasks processed by one consumer per iteration.
+local batch_size = 100
+
+local barrier = fiber.cond()
+local wait_count = 0
+
+box.cfg()
+
+local test_queue = queue.create_tube('test_queue', 'fifo', {temporary = true})
+
+local function prepare_consumers()
+    local consumers = {}
+
+    for i = 1, consumers_count do
+        consumers[i] = fiber.new(function()
+            wait_count = wait_count + 1
+            -- Wait for all consumers to start.
+            barrier:wait()
+
+            -- Create test tasks.
+            for j = 1, batch_size do
+                test_queue:put('test, cons: ' .. i)
```

Comment:
Maybe it's not completely fair, because some time was spent for concatenation. Maybe you could create target string only once and then reuse it several times.

Reply:
Updated.